typescript/sdk/src/contracts.ts

```diff
@@ -38,34 +44,31 @@ export function serializeContractsMap<F extends HyperlaneFactories>(
 export function serializeContracts<F extends HyperlaneFactories>(
   contracts: HyperlaneContracts<F>,
 ): HyperlaneAddresses<F> {
-  return objMap(
-    contracts,
-    (_, contract) => contract.address,
-  ) as HyperlaneAddresses<F>;
+  return objMap(contracts, (_, contract) => contract.address);
```

Comment:
Does this fix the casting + ts-ignoring we've had to do around this function? If so, happy to see it!

Reply:
I think so!